recipes/coremltools/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "coremltools" %}
+{% set version = "4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The sdist on PyPI lacks at least
+  #   - LICENSE.txt
+  #   - CMakeLists.txt
+  # so we use GitHub releases.
+  url: https://github.com/apple/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "b254b3b1bdb0d36ea83b34993d5de6d5ca3ab89a93e759d49e7a62307cad694c"
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - numpy
+  host:
+    - python
+    - numpy >=1.14.5
```

If this is using the numpy C API, see: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

```suggestion
    - numpy
```

If it isn't, this can probably be removed from `host`.

---

Thanks. I believe it does use the NumPy C API: https://github.com/apple/coremltools/blob/523d5e03d86c26267ee6bdf17dd20f6ce6bdadd7/CMakeLists.txt#L142-L152

---

Yup it definitely is, so you'll need the pin_compatible stuff 👍